backend/dataall/modules/mlstudio/api/resolvers.py

```diff
@@ -90,6 +100,31 @@ def delete_sagemaker_studio_user(
     )
 
 
+def create_sagemaker_studio_domain(context: Context, source, input: dict = None):
+    """Creates a SageMaker Studio user. Deploys the SageMaker Studio user stack into AWS"""
+    RequestValidator.validate_domain_creation_request(input)
+    return SagemakerStudioService.create_sagemaker_studio_domain(
+        uri=input["environmentUri"],
+        data=input
+    )
+
+
```

Comment:
Very minor: can we call the delete and create in the same way? `create_sagemaker_studio_domain` and `delete_sagemaker_studio_domain` or with `environment`

Reply:
Yes - the reason the delete and get methods contain `environment` was because when thinking of a future where ML Studio Domain CRUD operations are no longer coupled with environment operations (and if we have one env to many domains)

In that case there may be API calls to get/delete domain RDS records by environment URI (which is what we do now) - but also will likely need to get/delete domain RDS records by `sagemakerStudioDomainUri` which would more appropriately be named `delete_sagemaker_studio_domain`

Happy to change to keep it consistent though since I am not sure when (if ever) in the future we plan to take up the task of separating domains from environment creation - let me know your thoughts @dlpzx